contracts/axelar-gateway/src/contract.rs

```diff
@@ -76,9 +76,7 @@ impl AxelarGateway {
             minimum_rotation_delay,
             previous_signers_retention,
             initial_signers,
-        )?;
-
-        Ok(())
+        )
```

Comment:
you can add validate_proof as an entry point to the contract to make it easier to test authentication, might be helpful for on-chain debugging as well